packages/server/lib/automation/automation.ts

```diff
@@ -86,12 +86,12 @@ export class Automation {
       const onReq = this.get('onRequest')
 
       if (onReq) {
+        debug('Middleware `onRequest` fn found, attempting middleware exec for message: %s', message)
+
         return Bluebird.try(() => {
           return onReq(resolvedMessage, resolvedData)
         }).catch((e) => {
           if (AutomationNotImplemented.isAutomationNotImplementedErr(e)) {
```

Comment:
I think we missed this previously, but is it possible to add a test for the generic middleware here? If we think its out of scope here that's fine but I would like us to eventually add some more tests to the generic automation client.